event_model/__init__.py

```diff
@@ -390,9 +390,19 @@ class Filler(DocumentRouter):
     def __init__(self, handler_registry, *,
                  include=None, exclude=None, root_map=None,
                  handler_cache=None, resource_cache=None, datum_cache=None,
-                 descriptor_cache=None,
+                 descriptor_cache=None, inplace=None,
                  retry_intervals=(0.001, 0.002, 0.004, 0.008, 0.016, 0.032,
                                   0.064, 0.128, 0.256, 0.512, 1.024)):
+
+        if inplace is None:
+            self._inplace = True
```

Comment:
I like that this is internal (i.e. has an underscore). Should we have a public, read-only property so that the user can check whether a given `FIller` instance is in-place or not?

Reply:
added